### ISSUE TITLE
Improve install docs to remove this impass

### ISSUE THREAD

**Issue by eileenmcnaughton** [civicrm/civicrm-dev-docs#713]
https://github.com/totten/civix/pull/166

- the issue gives info as to what is needed

**Comment by MikeyMJCO** [comment]
We're working on splitting the installation docs out to a separate book - would you see this as part of that book or part of dev docs? 

For me separating it out makes more sense as the installation steps are a mixture of user/sysadmin and developer guide stuff and threading it between three books seems like a bad solution.

**Comment by totten** [comment]
Yeah, I agree that one doesn't want to split the extension-install steps between the different guides - they should live in one guide, probably with some kind of organization/subsections for the different techniques.

And I assume there be should cross-references? For example, if one were writing a developer "tutorial" on how create a new extension, then you might include a specific step for enabling the new extension, and you'd probably include a hyperlink to the deeper install documentation.

**Comment by MikeyMJCO** [comment]
Absolutely wrt cross referencing and the installation guide would be clearly sectioned into steps for:

* End User
* Sysadmin
* Developer
